docs/man/docker-create.1.md

```diff
@@ -80,7 +80,7 @@ IMAGE [COMMAND] [ARG...]
 **--cgroup-parent**=""
    Path to cgroups under which the cgroup for the container will be created. If the path is not absolute, the path is considered to be relative to the cgroups path of the init process. Cgroups will be created if they do not already exist.
 
-**--cpu-peroid**=0
+**--cpu-period**=0
```

Comment:
This is just a typo in the man page, not a misspelt command-line flag.

The flag is defined in [/api/client/build.go](https://github.com/docker/docker/blob/master/api/client/build.go) on line 57, and is spelt “period” not “peroid”.